tests/meta/demultiplex/test_demux_post_processing.py

```diff
@@ -203,7 +208,9 @@ def test_post_processing_tracks_undetermined_fastqs_for_bclconvert(
     # GIVEN a flow cell with undetermined fastqs in a non-pooled lane
 
     # WHEN post processing the flow cell
-    demux_post_processing_api.finish_flow_cell(bclconvert_flow_cell_dir_name)
+    demux_post_processing_api.finish_flow_cell(
+        flow_cell_directory_name=bclconvert_flow_cell_dir_name, bcl_converter=BclConverter.DRAGEN
```

Comment:
A bit confused, is BclConverter.DRAGEN when we run BCLConvert ON dragen and the BCLConverter.BCLConvert when we run it off-dragen? I think I have asked this in previous PRs as well but I am still confused about the difference.

Reply:
This is very confusing. There is an open issue to unify these constants into one but I decided to solve it after this PR is merged